src/class/cdc/cdc_host.c

```diff
@@ -886,7 +886,7 @@ static void acm_process_config(tuh_xfer_t* xfer) {
 
   switch (state) {
     case CONFIG_ACM_SET_CONTROL_LINE_STATE:
-      #if CFG_TUH_CDC_LINE_CONTROL_ON_ENUM
+      #ifdef CFG_TUH_CDC_LINE_CONTROL_ON_ENUM
```

Comment:
#if is not correct.
the value 0 for CFG_TUH_CDC_LINE_CONTROL_ON_ENUM is also valid to reset RTS and DTR (after reset of the host-CPU without re-power-up or re-plug of the USB CDC device).
Only if CFG_TUH_CDC_LINE_CONTROL_ON_ENUM is undefined, then this step should be skipped.